sqlup-mode.el

```diff
@@ -188,7 +187,7 @@ the given DIALECT of SQL."
   "Depending on the major mode (redis-mode or sql-mode), find the
 correct keywords. If not, create a (hopefully sane) default based on
 ANSI SQL keywords."
-  (cond ((sqlup-redis-mode-p) (mapcar 'downcase (sqlup-get-redis-keywords)))
+  (cond ((derived-mode-p 'redis-mode) (mapcar 'downcase (sqlup-get-redis-keywords)))
```

Comment:
I think it's better to keep a function here, instead of inlining `derived-mode-p`. I tried to write three arguments for why and I realized you could just as easily argue for this current  setup, so I'm curious as to why you decided to inline it.